xedocs/schemas/detector_numbers.py

```diff
@@ -15,10 +14,7 @@ class DetectorNumber(VersionedXeDoc):
 
     field: str = rframe.Index(max_length=80)
     partition: PARTITION = rframe.Index(default="all_tpc")
-
+    definition: str=""
     value: float
     uncertainty: float
-    definition: str
     reference: str = ""
-    date: datetime.datetime
-    comments: str = ""
```

Comment:
Is there any particular reason why we have to remove `comments`?

Reply:
We already have comments field in the parent class. Similarly with date.